template.php

```diff
@@ -117,7 +117,7 @@ function campaignion_foundation_preprocess_page(&$vars) {
   $vars['has_sidebar'] = $has_sidebar;
   $vars['is_narrow'] = $is_single_column || (!$has_sidebar && !$has_teasers);
   // Layout helper classes.
-  if ($vars['layout'] === 'cover-2col') {
+  if ($vars['layout'] === 'cover-2col' && $vars['page']['content_bottom']) {
```

Comment:
I guess this will need a null coalescing operator or `!empty()` as well.

Reply:
I didn’t think so because `$vars['page']` should definitely exist in this hook. But better be safe then sorry ;)

Reply:
`$vars['page']['content_bottom']` doesn’t need to exist at least not due to the hunk above in the same commit.